python_vrpn/README

```diff
@@ -12,7 +12,7 @@ tar xzvf python_vrpn.tar.gz
 cd python_vrpn
 edit Makefile.python and set the environment according to your setup
 make vrpn-python
-(as root) make install-vrpn-python
```

Comment:
Thanks for sending this.

The target is actually named install-vrpn-python, so the original command is correct.